Add info for 1.2 version of upgrade manager (#50)
neo4j/docs-ops-manager: modules/ROOT/pages/operation/upgrade-manager.adoc

= Upgrade manager
:description: This section describes the upgrade manager in Neo4j Ops Manager.

Upgrade manager supports you in upgrading or migrating your managed DBMSs. 

[IMPORTANT]
====

> 

Reviewer: ```suggestion
```

> The plans generated by NOM are intended as a guide to performing a Neo4j upgrade or migration. They are not a replacement for reading the documentation, upgrade planning and ensuring that you understand the purpose of all the steps.

Reviewer: ```suggestion
The plans generated by NOM are intended as a guide to performing a Neo4j upgrade or migration. 
They are not a replacement for reading the documentation, upgrade-planning, and ensuring that you understand the purpose of all the steps.
```

> 

Reviewer: ```suggestion
```

====

. Explore available versions
+ 
image::upgrades.png[width=800]
+

. Select a version and request a step by step tailored plan to guide you through the upgrade of the managed DBMS from it’s current version to the selected version.
Default settings (e.g. backup directory) can be overridden to suit your deploymemt environment and preferences. 
+ 
image::upgrades-choose-plan.png[width=800]
+

. Work through the upgrade plan - 
use the command line and cypher code snippet suggestions, follow links to documentation and check off the steps as you complete them.

> NOM will save your progress so you can take a break and continue with your upgrade later. 

Reviewer: ```suggestion
NOM saves your progress so you can take a break and continue with your upgrade later. 
```
